fbw-a380x/src/wasm/fadec_a380x/src/Fadec/Table1502_A380X.hpp

```diff
@@ -30,23 +30,22 @@ class Table1502_A380X {
    * @return A 2D array representing the CN2 - correctedN1 pairs.
    */
   static constexpr double table1502[13][4] = {
```

Comment:
The values in the comments here dont reflect the table values and seem copied from the a32nx

Reply:
see above - did not touch the values